locale/templates/auth.ftl

```diff
@@ -0,0 +1,305 @@
+
+automated-email-reset = This is an automated email; if you did not authorize this action, then <a data-l10n-name="resetLink">please reset your password</a>.
+  For more information, please visit <a data-l10n-name="supportLink">{ -brand-mozilla } Support</a>.
+# Variables:
+#  $resetLink (String) - Link to https://accounts.firefox.com/reset_password
+automated-email-reset-plaintext = If you did not change it, please reset your password now at { $resetLink }
+
+change-password-plaintext = If you suspect that someone is trying to gain access to your account, please change your password.
+
+# Variables:
+#  $ip (Number) - User's IP address
+user-ip = IP address: { $ip }
+
+manage-account = Manage account
+manage-account-plaintext = { manage-account }:
+
+subscriptionSupport = Questions about your subscription? Our <a data-l10n-name="subscriptionSupportUrl">support team</a> is here to help you.
+# After the colon, there's a link to https://accounts.firefox.com/support
+subscriptionSupport-plaintext = Questions about your subscription? Our support team is here to help you:
+
+subscriptionUpdatePayment = To prevent any interruption to your service, please <a data-l10n-name="updateBillingUrl">update your payment information</a> as soon as possible.
+# After the colon, there's a link to https://accounts.firefox.com/subscriptions
+subscriptionUpdatePayment-plaintext = To prevent any interruption to your service, please update your payment information as soon as possible:
+
+# Variables:
+#  $supportUrl (String) - Link to https://accounts.firefox.com/support
+support-message = For more information, please visit { $supportUrl }
+
+cadReminderFirst-subject = Your Friendly Reminder: How To Complete Your Sync Setup
+cadReminderFirst-action = Sync another device
+cadReminderFirst-title = Here’s your reminder to sync devices.
+cadReminderFirst-description = It takes two to sync. Syncing another device with { -brand-firefox } privately keeps your bookmarks, passwords and other { -brand-firefox } data the same everywhere you use { -brand-firefox }.
+
+cadReminderSecond-subject = Final Reminder: Complete Sync Setup
+cadReminderSecond-action = Sync another device
+cadReminderSecond-title = Last reminder to sync devices!
+cadReminderSecond-description = Syncing another device with { -brand-firefox } privately keeps your bookmarks, passwords and other { -brand-firefox } data the same everywhere you use { -brand-firefox }.
+
+# The user has a low number of valid recovery codes remaining for use
+codes-reminder-title = Low recovery codes remaining
+codes-reminder-description = We noticed that you are running low on recovery codes. Please consider generating new codes to avoid getting locked out of your account.
+codes-generate = Generate codes
+codes-generate-plaintext = { codes-generate }:
+lowRecoveryCodes-subject =
+    { NUMBER($numberRemaining) ->
+        [one] 1 recovery code remaining
+       *[other] { NUMBER($numberRemaining) } recovery codes remaining
```

Comment:
You should not need to use `NUMBER()` here, or at the beginning

```
lowRecoveryCodes-subject =
    { $numberRemaining ->
        [one] 1 recovery code remaining
       *[other] $numberRemaining recovery codes remaining
   }
```

Reply:
When I try what you suggested, the string literal "$numberRemaining" is printed for `*[other]*`:

```
Subject: $numberRemaining recovery codes remaining
```

It appears to work fine with this which I've gone with:
```
lowRecoveryCodes-subject =
    { $numberRemaining ->
        [one] 1 recovery code remaining
       *[other] { $numberRemaining } recovery codes remaining
   }
```